examples/cms-wordpress/.gitignore

```diff
@@ -0,0 +1,3 @@
+.next
```

Comment:
Can we drop this file?

Reply:
@Timer That can make it a little harder to work with for people that clone the example

Reply:
Doesn't Create Next App automatically inject this?